benchexec/tools/ultimate.py

```diff
@@ -72,7 +72,7 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
                 status = result.RESULT_UNKNOWN
                 break
             elif line.startswith('ERROR'):
-                status = 'ERROR'
+                status = line
```

Comment:
The result of this method should be a reasonably short status code (1-3 words) that makes sense to print in a table cell. Would returning the whole line here exceed this, or is it guaranteed to be short?